recipes/pyabel/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "PyAbel" %}
+{% set version = "0.7.4" %}
+{% set md5 = "1f9be2ecdc1ea8424000d62e2d6a4b10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy >=1.6
+    - setuptools >=16.0
+    - scipy >=0.14
+    - six >=1.10.0
```

Comment:
@jakirkham  Thaks for the review! I would say `cython`, `toolchain` is only needed at build time,  `setuptools`, `cython` is needed to run the `setup.py`, the rest  is imported when running this package. We `cimport numpy` from [a Cython script](https://github.com/PyAbel/PyAbel/blob/a4ecf6a39f607e9e15cd0d968d2956c12f4a2393/abel/lib/direct.pyx) so I imagine this does link against numpy? We are also linking against `libm.so` (to use `libc.math` from Cython), I'm not sure if needs to be added as an explicit dependency or if it is built-in?